pages/define-user-page.ts

```diff
@@ -0,0 +1,37 @@
+// Define user before installation
```

Comment:
let's remove this kind of comment in each file, should be obvious what is does, all the actions should be "before installation" as the installer behaves like a planner.